api/resolve/__init__.py

```diff
@@ -243,9 +156,11 @@ def get_version_conditions(version_name: str | None) -> list[str]:
             )
         """
         ]
+
     if version_name == "hero":
         return ["v.version < 0"]
 
+    logging.debug(f"Invalid version name: {version_name}")
```

Comment:
Due to logic above - if the input value would've been `vv001` it would strip the first v, making the reported invalid version name `v001`  even though the input value was `vv001`. We should maybe make sure we keep a copy of the input value and make sure to log the source input value.